includes/definitions/discovery/zynos.yaml

```diff
@@ -66,35 +115,62 @@ modules:
                     descr: 'Temp {{ $index }} ({{ $tempDescr }})'
                     index: 'tempCurValue.{{ $index }}'
                     high_limit: tempHighThresh
+        dbm:
+            options:
+                divisor: 100
+            data:
                 -
-                    oid: transceiverDdmInfoTable
-                    value: transceiverDdmInfoEntryCurrent
-                    snmp_flags: ['-OteQUsb', '-Pu'] # workaround for underscores in mib
-                    num_oid: '.1.3.6.1.4.1.890.1.5.8.47.117.2.1.7.{{ $index }}'
-                    descr: 'SFP ifIndex {{ $transceiverDdmInfoEntryPort }} {{ $transceiverDdmInfoEntryDescription }}'
-                    index: 'transceiverDdmInfoEntryCurrent.{{ $index }}'
-                    high_limit: transceiverDdmInfoEntryAlarmMax
-                    low_limit: transceiverDdmInfoEntryAlarmMin
-                    warn_limit: transceiverDdmInfoEntryWarnMax
-                    low_warn_limit: transceiverDdmInfoEntryWarnMin
-                    divisor: 100
-                    entPhysicalIndex: transceiverDdmInfoEntryPort
-                    entPhysicalIndex_measured: ports
```

Comment:
are you sure nothing is using this?

Reply:
This discovery has never worked for us, We use two different types of Zynos devices and they use a different set of MIBs. I am happy to leave this in as it should not interfere with my new SFP Dbm discovery.

Reply:
Added back so I will resolve this conversation